packages/neuron-wallet/src/utils/export-history.ts

```diff
@@ -1,110 +1,269 @@
 import fs from 'fs'
+import { promisify } from 'util'
 import sqlite3 from 'sqlite3'
 import { get as getDescription } from 'database/leveldb/transaction-description'
 import i18n from 'locales/i18n'
+import AssetAccountInfo from 'models/asset-account-info'
 import shannonToCKB from 'utils/shannonToCKB'
+import sudtValueToAmount from 'utils/sudt-value-to-amount'
+import BufferUtils from 'utils/buffer'
+import { ChainType } from 'models/network'
 
+const formatDatetime = (datetime: Date) => {
```

Comment:
We may want to consider using library such as `moment.js` to parse the date-time format. `moment.js` also supports localization.

Reply:
For the sake of security, we prefer not to use a third-party package.

Reply:
Then this would mean we need to reinvent a lot of wheels 🤔

Reply:
> Then this would mean we need to reinvent a lot of wheels 🤔

Yes, we have to do that. There're real-world attacks from famous third-party packages.

Ref: https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident

Reply:
We also need to have a check and see if `npm audit` can help here. https://blog.npmjs.org/post/173719309445/npm-audit-identify-and-fix-insecure#:~:text=npm%20audit%20is%20a%20new,and%20recommendations%20for%20further%20troubleshooting.

Reply:
> We also need to have a check and see if `npm audit` can help here. [blog.npmjs.org/post/173719309445/npm-audit-identify-and-fix-insecure#:~:text=npm%20audit%20is%20a%20new,and%20recommendations%20for%20further%20troubleshooting](https://blog.npmjs.org/post/173719309445/npm-audit-identify-and-fix-insecure#:~:text=npm%20audit%20is%20a%20new,and%20recommendations%20for%20further%20troubleshooting).

`npm audit` or `yarn upgrade-interactive` are not smart enough.

We have compared `Renovate`, `Snyk`, and `dependabot` and picked the last one as our dep bot who will push fixes timely.

E.g. https://github.com/nervosnetwork/neuron/pull/1700 and